src/Fuzzer/Testing/Generators/RandomInputGenerator.cs

```diff
@@ -0,0 +1,39 @@
+namespace Draco.Fuzzer.Testing.Generators;
+
+internal class RandomInputGenerator : IInputGenerator
```

Comment:
Maybe this could be a `RandomTextGenerator`, since a random token generator would also be a `RandomInputGenerator`